src/controllers/music/songs.js

```diff
@@ -8,197 +7,207 @@ import * as userSettings from '../../scripts/settings/userSettings';
 import globalize from '../../scripts/globalize';
 import '../../elements/emby-itemscontainer/emby-itemscontainer';
 import Dashboard from '../../utils/dashboard';
+import {playbackManager} from '../../components/playback/playbackmanager';
 
 /* eslint-disable indent */
```

Comment:
It looks like you fixed the indentation issues in this file so these comments can be removed.